### DISCUSSION TITLE
Add User-Agent setting

### DISCUSSION THREAD

**Discussion by Sustuso76** [CommunityToolkit/Maui#1526] [New Feature Discussions]
It would be interesting to have the ability to change the player's user-agent.

**Comment by CliffAgius** [comment]
Not sure what your looking for here but you can already set the UserAgent in the Webview if you check out the docs here:

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/webview?view=net-maui-8.0&pivots=devices-android

If this isn't what you was suggesting please reply with a little more detail or a use case.

![image](https://github.com/CommunityToolkit/Maui/assets/5613809/03da992a-babb-48dd-b018-dcbc5a9b3939)

**Reply by GeorgeLeithead** [reply]
As @CliffAgius has provided, I think that this is already in-place.

**Reply by bijington** [reply]
The use of the word player in the original post makes me think they want to change it for the `MediaElement`. I can't comment on why though...

**Reply by Sustuso76** [reply]
I would like to know with which useragent my app requests a video, and to be able to change it if necessary.

Or if there is a way to change the user agent for the entire application (and therefore I assume also that of the CommunityToolkit).

I'm not interested in changing the useragent of a WebKit component present in a page, but either that of the CommunityToolkit or of the entire application.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers.
